src/App.js

```diff
@@ -1,5 +1,41 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { MESSAGES } from "../src/constants/messages.js";
+import carInputValidation from "./utils/carInputValidation.js";
+import countInputValidation from "./utils/countInputValidation.js";
+import progress from "./utils/progress.js";
+import racePrint from "./utils/racePrint.js";
+import winner from "./utils/winner.js";
 class App {
-  async play() {}
+  async play() {
+    MissionUtils.Console.print(MESSAGES.START);
+    const CARS_NAMES = await MissionUtils.Console.readLineAsync("");
+    carInputValidation(CARS_NAMES);
+
+    MissionUtils.Console.print(MESSAGES.ANSWER_COUNT);
+    const COUNT = await MissionUtils.Console.readLineAsync("");
+    countInputValidation(COUNT);
+
+    //게임 진행
+    MissionUtils.Console.print("\n" + MESSAGES.RESULT);
+    const CARS = CARS_NAMES.split(",");
+
+    const RACE = {};
+    CARS.forEach((CAR) => {
+      RACE[CAR.trim()] = 0;
+    });
```

Comment:
```suggestion
    const RACE = CARS.reduce((acc, car) => {
       acc[car.trim()] = 0;
       return acc;
    }, {});
```
기존 방식은 `RACE`라는 빈 객체를 만들고 자동차 목록을 순회하면서 0으로 초기화 해주셨는데, `reduce` 함수를 사용하면 따로 초기 값을 설정하지 않고, `reduce` 함수의 초기 값을 사용하는 방식으로도 가능할 것 같아요!

저도 예전에는 소은님 방식으로 많이 사용했던 것 같은데, 요즘에는 `reduce`를 사용해서 선언적으로 코드를 작성하고, 불변성을 유지하려고 하는 것 같아요🙃

Reply:
reduce를 잘 사용하지 않아서 제대로 몰랐는데 한번 써봐야겠네요 !